src/definitions/person.ts

```diff
@@ -26,9 +26,9 @@ export type PersonDefinitions = LocaleEntry<{
   suffix: string[];
 
   /**
-   * A list of patterns used to generate names.
+   * A weighted list of patterns used to generate names.
    */
-  name: string[];
+  name: { value: string; weight: number }[];
```

Comment:
I prefer 
```suggestion
  name: Array<{ value: string; weight: number }>;
```

But we can also turn on an eslint rule for that